behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -88,4 +103,40 @@ def step_impl_update_result_page(context, location, filter_label):
 def step_impl_error_shown_on_page(context):
     alert = context.helperfunc.find_by_css_selector(".alert-message")
     assert alert is not None
-    assert alert.text, "No results"
+    assert (
+        alert.text == "No results\nThere are no results matching your search criteria."
+    )
+
+
+@step(
+    'I am taken to the page corresponding to the "{location}" "{organisation}" search result'
+)
+def step_impl_result_page_with_multi_params(context, location, organisation):
+    result_number_paragraph = context.helperfunc.find_by_xpath(
+        '//p[@class="govuk-body"]'
+    )
+    current_url = context.helperfunc.driver().current_url
+    title_xpath = context.helperfunc.find_by_xpath(
+        "//html/body/div/main/div/div/h1"
+    ).text.replace("\n", " ")
+    result_container_xpath = context.helperfunc.find_by_xpath(
+        '//div[@class="search-results-container"]'
+    )
+    organisation_url_string = organisation.replace(" ", "+")
+    location_url_string = location.replace(" ", "+")
+
+    assert (
```

Comment:
Is the parenthesis intentional for this assert?

Reply:
the parenthesis has been added by the pre-commit linting hooks. The line is too long when placed on the same line.